Makefile

```diff
@@ -51,6 +51,12 @@ endif
 
 CROSS_ARCH=
 
+ifneq ($(CROSS_ARCH),)
```

Comment:
Shouldn't this just be passed as `TARGET` instead?

I imagine you'll get a few HOST == TARGET checks instead of CROSS_ARCH == "", but that seems okay.

Reply:
done.